app/container/volumes/template.hbs

```diff
@@ -20,7 +20,7 @@
         </td>
         <td data-title={{t 'containersPage.containerPage.volumesTab.table.body.shared'}}>
           <ul class="list-unstyled">
-          {{#each mount.volume.sharedWith as |container|}}
+          {{#each mount.sharedContainers as |container|}}
```

Comment:
@vincent99 i excluded 'self mount' from the list here https://github.com/rancher/ui/pull/764/files#diff-51d1b78fa8b9057349d954c5f62380d2R70